README.md

```diff
@@ -12,9 +12,10 @@
       src="https://cdn.rawgit.com/webpack/media/e7485eb2/logo/icon.svg">
   </a>
   <h1>UglifyJS Webpack Plugin</h1>
-	<p>This plugin uses <a href="https://github.com/mishoo/UglifyJS2/tree/v2.x">UglifyJS v2</a> to minify your JavaScript.<p>
```

Comment:
The real solution would be to figure out a nice way at **webpack-defaults** and then apply that across projects. There was some talk about using **markdown-magic** and templating.